plugins/kubelet_logs.yaml

```diff
@@ -0,0 +1,60 @@
+version: 0.0.1
+title: Kubernetes Kubelet
+description: Log parser for Kubelet journald logs
+parameters:
+  - name: journald_directory
+    description: Directory containing journal files to read entries from
```

Comment:
I think we should mention that this is optional, and the receiver will attempt to detect the correct directory.